uint32 outer_tag_id
uint32 outer_tag_valid_detections
geometry_msgs/Point outer_tag_position_global
geometry_msgs/Point outer_tag_position_local
geometry_msgs/Quaternion outer_tag_orinentation_local
float64 outer_tag_yaw
uint32 inner_tag_id
uint32 inner_tag_valid_detections
geometry_msgs/Point inner_tag_position_global
geometry_msgs/Point inner_tag_position_local
geometry_msgs/Quaternion inner_tag_orinentation_local
float64 inner_tag_yaw
geometry_msgs/Point reference_position
float64 reference_yaw
uint32 mpc_pose_sent_flag